### PR TITLE
My attempt

### PR DESCRIPTION
**Pull request by SammyAbukmeil**
Unable to get a functional game running unfortunately. Will continue when time allows.

### CODE REVIEW

File: views/play.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1>Rock Paper Scissors</h1>
+
+<p>Result</p>
+
+<form action="/play_again" method="POST">
```
Comment (tansaku) [inline views/play.erb]: POST should be lower case

File: views/new_game.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h1>Rock Paper Scissors</h1>
+
+<h2>Register to play:</h2>
+
+<form action="/first_pick" method="POST">
```
Comment (tansaku) [inline views/new_game.erb]: prefer lower case POST

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Game
+	attr_accessor :player_one, :player_two
```
Comment (tansaku) [inline lib/game.rb]: avoid attr_accessor where possible

File: lib/player.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_reader :choice
+  attr_accessor :name
```
Comment (tansaku) [inline lib/player.rb]: avoid attr_accessor where possible

**Comment by tansaku** [comment]
Hi Sammy,

Good work getting the basic game working. I was very pleased to see:

* good test coverage 
* rspecs passing
* good game logic in game
* the views looking good (minimal logic)
* the controllers looking good (minimal logic)

however you do have the following problems:

* hound/rubocop issues
* cucumber failing 
* avoid leaving empty files in the system: views/play_again.erb

It looks like you ran out of time hooking up the web version.  Do you need help moving forward?

:construction:

**Comment by SammyAbukmeil** [comment]
Sam,

Thanks for the great feedback, I should be fine moving forward but will seek help if I do encounter any road bumps!

**Pull request closed without merging**